frontend/src/components/ViewActivities/activity.js

```diff
@@ -80,7 +73,9 @@ class Activity extends React.Component {
           <Accordion.Toggle as={Card.Header} eventKey='0' align='center'>
             {activity[DB.ACTIVITIES_TITLE]}
           </Accordion.Toggle>
-          <Accordion.Collapse eventKey='0' className={'view-activity' + (this.state.editing? ' edit': '')}>
+          <Accordion.Collapse eventKey='0' 
+            className={'view-activity' + (this.state.editing? ' edit': '')}
+            >
```

Comment:
Very small nit: something I mentioned in one of the later PRs about the style Keiffer used for this (that I also stole). Up to you if you want to use it.
```suggestion
          <Accordion.Collapse 
            eventKey='0' 
            className={'view-activity' + (this.state.editing? ' edit': '')}
          >
```